tdrs-backend/zap-scanner.sh

```diff
@@ -8,27 +8,27 @@ fi
 
 docker-compose down
 docker-compose up -d --build
-	# do an OWASP ZAP scan
-	CONTAINER=$(docker-compose images | awk '/zaproxy/ {print $1}')
- 	export ZAP_CONFIG=" \
-	  -config globalexcludeurl.url_list.url\(0\).regex='.*/robots\.txt.*' \
-	  -config globalexcludeurl.url_list.url\(0\).description='Exclude robots.txt' \
-	  -config globalexcludeurl.url_list.url\(0\).enabled=true \
-	  -config spider.postform=true"
+    # do an OWASP ZAP scan
+     export ZAP_CONFIG=" \
+      -config globalexcludeurl.url_list.url\(0\).regex='.*/robots\.txt.*' \
+      -config globalexcludeurl.url_list.url\(0\).description='Exclude robots.txt' \
+      -config globalexcludeurl.url_list.url\(0\).enabled=true \
+      -config spider.postform=true"
 
-	echo "================== OWASP ZAP tests =================="
-	docker exec "$CONTAINER" zap-full-scan.py -t http://web:8080/ -m 5 -z "${ZAP_CONFIG}" | tee /tmp/zap.out 
-	if grep 'FAIL-NEW: 0' /tmp/zap.out >/dev/null ; then
-		ZAPEXIT=0
-	else
-		ZAPEXIT=1
-	fi
+    echo "================== OWASP ZAP tests =================="
+    chmod 777 $(pwd)/reports
+    docker-compose run zaproxy zap-full-scan.py \
```

Comment:
`docker exec` won't write the needed html file, so we used `docker-compose run` instead